crates/services/upgradable-executor/wasm-executor/src/preconfirmation_sender.rs

```diff
@@ -0,0 +1,14 @@
+use fuel_core_executor::ports::PreconfirmationSenderPort;
+use fuel_core_types::services::preconfirmation::PreconfirmationStatus;
+
+pub struct PreconfirmationSender;
```

Comment:
What's the difference between this and `TransparentPreconfirmationSender`

Reply:
There is not difference, I think @AurelienFT just wanted to define type explicitly in the WASM executor in the case if someone will update logic of `TransparentPreconfirmationSender` in the future

Reply:
Yeah I tried to follow what we do for relayer and thinks like that have something defined only for wasm executor